src/dlgConnectionProfiles.cpp

```diff
@@ -297,6 +300,7 @@ void dlgConnectionProfiles::slot_update_discord_optin(int state)
 
     if (state == Qt::Checked) {
         pHost->mDiscordDisableServerSide = false;
+        // FIXME: this requests GMCP data to get Discord data but that is NOT a requirement to use DISCORD!
```

Comment:
That has now been fixed - but if GMCP *has* been disabled we might want to consider the alternative MSDP when that gets sorted out again so it can work...